packages/rest/src/lib/RequestManager.ts

```diff
@@ -404,14 +407,25 @@ export class RequestManager extends EventEmitter {
 			}
 		}
 
-		const fetchOptions = {
-			agent: this.agent,
-			body: finalBody,
+		const resolvedBody = await resolveBody(finalBody);
+
+		if (Array.isArray(resolvedBody)) {
+			finalBody = resolvedBody[0]; // FormData stringified
+			additionalHeaders['content-type'] = resolvedBody[1]; // multipart header
+		} else {
+			finalBody = resolvedBody;
+		}
+
+		const fetchOptions: RequestOptions = {
 			// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
 			headers: { ...(request.headers ?? {}), ...additionalHeaders, ...headers } as Record<string, string>,
-			method: request.method,
+			method: `${request.method}` as Dispatcher.HttpMethod,
```

Comment:
Any reason for `${}`? Can't you just cast it?

Reply:
Left overs from before I changed the enum to be uppercase